paddle/contrib/dynamic/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
I think `tape` is a good namespace and directory name than `dynamic`, because `dynamic` is simply incomplete -- what we want to say is `dynamic_net`, which, however, is too long as a namespace name.